t/react_template/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "react-template",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
+    "@mui/icons-material": "^5.11.11",
+    "@mui/material": "^5.11.13",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^16.18.11",
+    "@types/react": "^18.0.26",
+    "@types/react-dom": "^18.0.10",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "typescript": "^4.9.4",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eslint:lint": "eslint 'src/**/*.{ts,tsx}'",
```

Comment:
We no longer use separated linter, gts could handle all of them for us. Instead of having separated cmd here, you can do
1) only use https://github.com/google/gts
2) use [.prettierrc.json](https://github.com/google/gts/blob/main/.prettierrc.json) to customize underlying configs

Reply:
removed eslint